File: APVSolucion/APV/conexion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using MySql.Data.MySqlClient;

namespace APV
{
    class conexion
    {
        public MySqlConnection cn = new MySqlConnection("server=127.0.0.1;database=puntoventa;Uid=root;pwd=;");

        public MySqlDataReader dr;
             
        public void abrir()
        {
            cn.Open();
        }

        public void cerrar()
        {
            cn.Close();
        }

        public void movimientos(string query)
        {
            MySqlCommand cmd = new MySqlCommand(query, cn);
            cmd.ExecuteNonQuery();
        }

        public void consulta(string query)
        {
            MySqlCommand cmd = new MySqlCommand(query, cn);
            dr = cmd.ExecuteReader();
        }
    }
}

File: DatabaseVB/DatabaseVB/conexion.vb
Imports MySql.Data.MySqlClient
Imports System.Data

Public Class conexion
    Public cn As MySqlConnection = New MySqlConnection("server=127.0.0.1; database=prueba; Uid=root; pwd=;")
    Public dr As MySqlDataReader

    Public Sub abrir()
        cn.Open()
    End Sub

    Public Sub cerrar()
        cn.Close()
    End Sub

    Public Sub consulta(ByVal query As String)
        Dim cmd As MySqlCommand = New MySqlCommand(query, cn)
        dr = cmd.ExecuteReader()
    End Sub

    Public Sub movimientos(ByVal query As String)
        Dim cmd As MySqlCommand = New MySqlCommand(query, cn)
        cmd.ExecuteNonQuery()
    End Sub
End Class
